.github/workflows/pr-smoke-test.yml

```diff
@@ -96,7 +96,7 @@ jobs:
             await script({github, context})
 
       - name: Remove label from pull request.
-        if: always() && ${{ contains(github.event.label.name, format( 'run{0} smoke tests', ':') ) }}
+        if: always() && ${{ contains( github.event.pull_request.labels.*.name, format('run{0} smoke tests', ':')) }}
```

Comment:
We're using `always()` here so that we can kick off the smoke tests at any point, not just when opening a pull request?  That seems fine, but I just noticed it as a difference between the test code and what we've got here.

Reply:
Cool, thanks for educating me on that!